misc/python/materialize/query_fitness/pick.py

```diff
@@ -10,16 +10,16 @@
 
 import sys
 
+import pg8000
 import sqlparse
-from pg8000.dbapi import Connection  # type: ignore
 
 from materialize.query_fitness.all_parts_essential import AllPartsEssential
 
 threshold = 0.5
 
 
 def main() -> None:
-    conn = Connection("pstoev", password="pstoev")
+    conn = pg8000.connect(database="pstoev", password="pstoev")
```

Comment:
@philip-stoev this seems mighty suspicious. Is this supposed to hardcoded your name?

Reply:
Yes I have a fix for that locally.